app/marketing/mails.py

```diff
@@ -35,15 +37,24 @@
     render_start_work_approved, render_start_work_new_applicant, render_start_work_rejected, render_tip_email,
 )
 from sendgrid.helpers.mail import Content, Email, Mail, Personalization
+from sendgrid.helpers.stats import Category
+
+
+def func_name():
+    return inspect.stack()[0][3]
```

Comment:
I only say that because we're running inspect in a production environment. If we ever opted to move away from CPython for whatever reason, it would bomb out. Also, I mention inspect.currentframe().f_code.co_name versus inspect.stack()[0][3] because it's typically MUCH faster to run the currentframe call over stack.
Also, will this ultimately return func_name for all calls or the parent method name (`func_name`)?

Reply:
As @mbeacom explained it over slack with this
```
import inspect


def func_name():
    return inspect.stack()[0][3]

def derp():
    print(func_name())
```